lib/tasks/create_tags.rake

```diff
@@ -55,19 +55,106 @@ namespace :tags do
     puts '-> Sequel_16_barcodes_v3 tags successfully created'
 
     puts
-    puts '-> Creating ONT tag set for 96 sample wells'
-    constants_accessor = Pipelines::ConstantsAccessor.new(Pipelines.ont.covid)
-    tag_set = TagSet.create!(name: constants_accessor.pcr_tag_set_name, uuid: SecureRandom.uuid)
-    puts '-> Tag Set successfully created'
-    oligo = +'ACGTACGTACGTACGT'
-    (1..96).each do |tag_index|
-      padded_tag_number = format('%<tag_number>02i', { tag_number: tag_index })
-      Tag.create!(oligo: oligo,
-                  group_id: "ont_96_tag_#{padded_tag_number}",
-                  tag_set_id: tag_set.id)
-      oligo.increment_oligo!
```

Comment:
I will hate to see it go, but if we don't need to increment the oligo any more, the extension methods at the top of this file can be thrown away 😞

Reply:
We can keep this. I'm adding a new rake task to seed the same tag set using data fetched from sequencescape